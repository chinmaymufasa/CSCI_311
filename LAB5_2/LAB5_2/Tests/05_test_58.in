2
94
5
2
132
5
2
34
5
2
75
4
2
61
6
2
78
4
2
53
6
9
3
34
6
9
2
122
5
2
145
6
9
3
94
6
9
2
159
4
2
159
5
2
55
6
9
3
145
6
9
2
143
4
2
81
4
2
137
5
2
87
4
2
157
5
2
99
6
2
70
6
9
3
772
6
9
2
159
4
-1
