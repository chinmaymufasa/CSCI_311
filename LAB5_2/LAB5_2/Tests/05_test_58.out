94 
94 132 
34 94 132 
94 34 75 132 
34 75 61 132 94 
75 61 34 94 78 132 
34 61 53 78 132 94 75 
7
61 53 78 132 94 75 
6
53 61 75 78 94 122 132 
61 53 78 122 145 132 94 75 
8
61 53 78 145 132 122 75 
7
75 53 61 122 78 145 132 159 
53 61 75 78 122 132 145 159 159 
53 61 55 78 132 122 159 159 145 75 
10
53 61 55 78 132 122 159 159 75 
9
75 55 53 61 132 122 78 159 143 159 
75 55 53 61 132 81 78 122 159 143 159 
53 55 61 75 78 81 122 132 137 143 159 159 
132 75 55 53 61 81 78 122 87 159 143 137 159 
53 55 61 75 78 81 87 122 132 137 143 157 159 159 
53 61 55 78 87 122 99 81 75 137 157 143 159 159 132 
53 70 61 55 78 87 122 99 81 75 137 157 143 159 159 132 
16
53 70 61 55 78 87 122 99 81 75 137 157 143 159 159 132 
16
132 75 55 53 61 70 81 78 99 87 122 157 143 137 159 159 159 
